aspnetcore/security/authentication/cookie.md

```diff
@@ -210,17 +210,17 @@ If you aren't using `CookieAuthenticationDefaults.AuthenticationScheme` (or "Coo
 
 Once a cookie is created, it becomes the single source of identity. Even if you disable a user in your back-end systems, the cookie authentication system has no knowledge of this, and a user stays logged in as long as their cookie is valid.
 
-The `ValidateAsync()` event can be used to intercept and override validation of the cookie identity. This approach mitigates the risk of unauthorized access to an app.
+The [ValidatePrincipal](/dotnet/api/microsoft.aspnetcore.authentication.cookies.cookieauthenticationevents.validateprincipal) event in ASP.NET Core 2.x or the [ValidateAsync](/dotnet/api/microsoft.aspnetcore.identity.isecuritystampvalidator.validateasync) method in ASP.NET Core 1.x can be used to intercept and override validation of the cookie identity. This approach mitigates the risk of unauthorized access to an app.
```

Comment:
ValidatePrincipal is the same method in 1.x and 2.x. ISecurityStampValidator is called inside of Identity's implementation of ValidatePrincipal.

Reply:
It's `ValidatePrincipal` vs. `ValidateAsync` tho, which reflects that the 2.x sample is using `ValidatePrincipal`, while I left the 1.x sample using the `ValidateAsync` unchanged.